src/simulation.cpp

```diff
@@ -105,7 +106,16 @@ int openmc_simulation_init()
     load_state_point();
     write_message("Resuming simulation...", 6);
   } else {
-    initialize_source();
+	//Only initialize primary source bank for eigenvalue simulations
+	if (settings::run_mode == RunMode::EIGENVALUE) {
+	  initialize_source();
+	}
+  }
+
+  // If fixed source and using custom source library then need to load
+  if (settings::run_mode == RunMode::FIXED_SOURCE &&
+        !settings::path_source_library.empty()) {
+	load_custom_source_library();
   }
```

Comment:
Use two spaces per indent. I couldn't find mention in the OpenMC Style Guide for C++ about the indent for continuation lines, but elsewhere in the code it looks like the continuation line is aligned with the first character after the opening parenthesis on the preceding line.

```suggestion
  if (settings::run_mode == RunMode::FIXED_SOURCE &&
      !settings::path_source_library.empty()) {
    load_custom_source_library();
  }
```